maestro-test/src/test/resources/088_launchApp_with_all_permissions_to_deny_and_notification_to_allow.yaml

```diff
@@ -0,0 +1,6 @@
+appId: com.example.app
+---
+- launchApp:
+    permissions:
+      all: deny
+      notifications: allow
```

Comment:
Shouldn't this be: `permissions: { all: deny, notifications: allow }`?

Reply:
You can write it either way..